### ISSUE TITLE
Write documentation on python x feel++ modules

### ISSUE THREAD

**Issue by thomas-saigre** [feelpp/book.feelpp.org#151]

**Comment by thomas-saigre** [comment]
Actually, I continue to write doc about the module I'm developing (and sometimes of other modules I use that are not yet documented on the website. I put those commits on the branch [saigre/pyfeelpp](https://github.com/feelpp/book.feelpp.org/commits/saigre/pyfeelpp), but in the commit message I forget to mention this issue...

**Comment by prudhomm** [comment]
@thomas-saigre is it still relevant ?

**Comment by thomas-saigre** [comment]
@prudhomm I don't think so, I opened this issue when I was developing the reduced_basis Python module, which was aborted.
